Reaqtive/Core/Reaqtive.Linq/Reaqtive/Subscribable.cs

```diff
@@ -927,9 +945,84 @@ public static ISubscribable<TSource> StartWith<TSource>(this ISubscribable<TSour
 
         #region Standard Query Operators
 
-        //
-        // TODO: Last, Single, DefaultIfEmpty
-        //
+        #region Contains
+
+        /// <summary>
+        /// Propagates a boolean value determining whether the source sequence contains specified element or not using default comparer for elements.
```

Comment:
Nit about doc comments: `the specified element`, `the default comparer`, etc. Can you check the comments below as well to make sure we're consistently using `the`.